src/core/scene/a-scene.js

```diff
@@ -547,9 +547,8 @@ module.exports.AScene = registerElement('a-scene', {
         renderer = this.renderer = new THREE.WebGLRenderer(rendererConfig);
         renderer.setPixelRatio(window.devicePixelRatio);
         renderer.sortObjects = false;
-        renderer.vr.setPoseTarget(this.camera);
```

Comment:
We need this line but it has to be `renderer.vr.setPoseTarget(self.camera.el.object3D);` to be consistent with the one below